app/src/main/java/org/fedorahosted/freeotp/add/ScanActivity.java

```diff
@@ -136,11 +137,10 @@ protected void onCreate(Bundle savedInstanceState) {
         fotoapparat = Fotoapparat
                 .with(this)
                 .into(cameraView)
-                .previewScaleType(ScaleType.CENTER_CROP)
-                .photoSize(biggestSize())
```

Comment:
Need to see what this can be, we have `photoResolution` and `previewResolution` available separately now, and values taken is `highestResolution`